src/main/java/org/logstash/dissect/fields/AppendField.java

```diff
@@ -0,0 +1,52 @@
+package org.logstash.dissect.fields;
+
+import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
+
+import java.util.Map;
+
+public final class AppendField extends AbstractField {
+    private static final int ORD = 2;
+
+    private AppendField(String s, int ord) {
+        super(s, ord);
+    }
+
+    public static Field create(String s) {
+        if (hasOrdinal(s)) {
+            String[] parts = s.split("/");
```

Comment:
There may be more than two elements - but I'm only interested in the first two.
I can split with `"\\W|_"`. Better?